packages/form-js-editor/src/render/components/FormEditor.js

```diff
@@ -136,6 +140,8 @@ function Element(props) {
         }
       </ContextPad>
       { props.children }
+      <FieldResizer position="left" field={ field }></FieldResizer>
```

Comment:
Wonder if we can somehow conditionally display this one based on position, to not display in cases where we don't want it.

Reply:
![firefox_GtTQpbMboA](https://user-images.githubusercontent.com/17801113/232431238-5aa4cbb4-e38f-4aa9-8ac9-c9ea80e1ccec.gif)

Lack of auto does make the usage a little strange I would say, though I'm not so sure what the behaviour we want is.

Reply:
That was mentioned via https://github.com/bpmn-io/form-js/pull/593#pullrequestreview-1385447889 already (for one case). I agree that the resizers do not make super sense for each case, but leaving them out because _we_ believe they are not useful is maybe not the best solution.

Probably a good indicator for waiting on feedback /cc @RomanKostka